unified_targets/configs/IFLIGHTF7_TWIN_G.config

```diff
@@ -0,0 +1,246 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.3 Jun  1 2019 / 11:59:57 (094cfc956) MSP API: 1.41
+
+board_name IFF7_TWIN_G
+manufacturer_id IFRC
+
+# resources
+resource BEEPER 1 C15
+resource MOTOR 1 C08
+resource MOTOR 2 C06
+resource MOTOR 3 C09
+resource MOTOR 4 C07
+resource MOTOR 5 B06
+resource MOTOR 6 B07
+resource MOTOR 7 B01
+resource MOTOR 8 B00
+resource SERVO 1 NONE
```

Comment:
The lines containing `NONE` and the settings below that set defaults should not be in here - they make the target more likely to break on future firmware changes.
To fix this, apply the changes to a Unified Target firmware, and then supply the output of `diff bare` as the Unified Target configuration.

Reply:
OK. I fixed them just now. It's time to merge.